setup.py

```diff
@@ -10,8 +10,8 @@
 from setuptools import Extension, setup
 
 
-if sys.version_info < (3, 5, 3):
-    raise RuntimeError("aiohttp 3.x requires Python 3.5.3+")
+if sys.version_info < (3, 6):
+    raise RuntimeError("aiohttp 3.x requires Python 3.6+")
```

Comment:
But if you want to keep it:
```suggestion
    raise RuntimeError("aiohttp 4.x requires Python 3.6+")
```